docs/content/preview/admin/yb-admin.md

```diff
@@ -1793,6 +1793,30 @@ yb-admin \
 * *replication_name*: The name of the replication to be enabled or disabled.
 * `0` | `1`: Disabled (`0`) or enabled (`1`). Default is `1`.
 
+#### change_xcluster_role
+
+Sets the xCluster role to `STANDBY` or `ACTIVE`.
+
+**Syntax**
+
+```sh
+yb-admin \
+    -master_addresses <target_master_addresses> \
```

Comment:
Here <target_master_addresses> is confusing, as user can run this command to source universe with source master addresses as well.
So nice to use as <master_addresses> and mention the details in description - master_addresses are the addresses of the master nodes where the role has to be applied(example if we want to change target to STANDBY we have to use target universe master addresses, and if we want to change source universe role then we have to use source universe master addresses) cc: @rahuldesirazu @hari90 @lingamsandeep

Reply:
Committed this change. Please recheck.